ArcGISRuntimeSDKQt_CppSamples/Routing/RouteAroundBarriers/RouteAroundBarriers.qml

```diff
@@ -139,23 +138,28 @@ Item {
                 Layout.margins: 3
                 color: "lightgrey"
 
-                ListView {
-                    id: directionsView
-                    anchors {
-                        fill: parent
-                        margins: 5
-                    }
-                    header: Component {
-                        Text {
-                            height: 40
-                            text: "Directions:"
-                            font.pixelSize: 22
+                ScrollView {
+                    width: parent.width
+                    height: parent.height
+
+                    ListView {
+                        id: directionsView
+                        anchors {
+                            fill: parent
+                            margins: 5
+                        }
+                        header: Component {
+                            Text {
+                                height: 40
```

Comment:
Should a height be set?

Reply:
Adding a height prevents the overlap between the header and the listview items that is seen here 
![image](https://user-images.githubusercontent.com/60744198/80352238-7be98080-886b-11ea-8b6f-2858f095d251.png)

Reply:
but this can also be avoided by adding padding, so I will do that instead